code/game/objects/items/airlock_painter.dm

```diff
@@ -114,15 +136,19 @@
 /obj/item/airlock_painter/examine(mob/user)
 	. = ..()
 	if(!ink)
-		. += "<span class='notice'>It doesn't have a toner cartridge installed.</span>"
+		. += "<span class='notice'>The ink compartment hangs open.</span>"
 		return
 	var/ink_level = "high"
-	if(ink.charges < 1)
-		ink_level = "empty"
-	else if((ink.charges/ink.max_charges) <= 0.25) //25%
-		ink_level = "low"
-	else if((ink.charges/ink.max_charges) > 1) //Over 100% (admin var edit)
-		ink_level = "dangerously high"
+	switch(ink.charges/ink.max_charges)
+		if(0.001 to PAINTER_LOW)
+			ink_level = "extremely low"
+		if(PAINTER_LOW to PAINTER_MID)
+			ink_level = "low"
+		if(1 to INFINITY) //Over 100% (admin var edit)
+			ink_level = "dangerously high"
+		else
+			ink_level = "empty"
```

Comment:
It's going to show empty for full ink cartridges

Reply:
is there aby way to test for 0 to negative infinitry